src/components/app/ServiceWorkerManager.js

```diff
@@ -82,56 +82,61 @@ class ServiceWorkerManagerImpl extends PureComponent<Props, State> {
     updatedWhileNotReady: false,
   };
 
+  _workbox: Workbox | null = null;
+
   _installServiceWorker() {
-    const runtime = require('offline-plugin/runtime');
-    runtime.install({
-      onInstalled: () => {
-        console.log('[ServiceWorker] App is ready for offline usage!');
-      },
-      onUpdating: () => {
-        // XXX Strangely, this doesn't seem to be called...
-        console.log(
-          '[ServiceWorker] An update has been found and the browser is downloading the new assets.'
-        );
-      },
-      onUpdateReady: () => {
-        console.log(
-          '[ServiceWorker] We have downloaded the new assets and we are ready to go.'
-        );
-        this.setState({
-          installStatus: 'pending',
-          isNoticeDisplayed: true,
-        });
-      },
-      onUpdated: () => {
-        // The update could have been triggered by this tab or another tab.
-        // We distinguish these cases by looking at this.state.installStatus.
-        console.log(
-          '[ServiceWorker] The new version of the application has been enabled.'
-        );
+    if (!('serviceWorker' in navigator)) {
+      return;
+    }
```

Comment:
We didn't have the check before but this is in workbox' examples and I thought this is a good idea.